scripts/migrations/anonymous_store_remediation.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+"""
+Identifies and deletes any remaining `account-email` store entries that are associated
+with anonymized accounts.
+"""
+import argparse
+from pathlib import Path
+
+import db
+import web
+
+import infogami
+from openlibrary.config import load_config
+
+DEFAULT_CONFIG_PATH = "/olsystem/etc/openlibrary.yml"
+
+def setup(config_path):
+    if not Path(config_path).exists():
+        raise FileNotFoundError(f'no config file at {config_path}')
+
+    load_config(config_path)
+    infogami._setup()
+
+def remediate(test=False):
+    def fetch_usernames():
+        oldb = db.get_db()
+
+        query = """
+            SELECT SUBSTRING(key FROM 9) as username from thing
+            WHERE type IN (SELECT id FROM thing WHERE key = '/type/delete' LIMIT 1)
+            AND key LIKE '/people/%'
+            AND LENGTH(key) - LENGTH(REPLACE(key, '/', '')) = 2
```

Comment:
```suggestion
            AND LENGTH(key) - LENGTH(REPLACE(key, '/', '')) = 2
```
This checks that the key only contains two `/` characters.  We do this to avoid fetching lists, preferences, and other pages.